### PR TITLE
kaldi: init at 2020-12-26

### PR DESCRIPTION
**Pull request by Mic92**
State-of-the-art speech-to-text engine - used in rhasspy

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Mic92** [comment]
I am using this package for 1-2 years.

File: pkgs/tools/audio/kaldi/default.nix [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+  buildInputs = [
+    openblas
+    openfst
+    icu
+  ];
+
+  nativeBuildInputs = [
+    cmake
+    pkgconfig
```
Comment (SuperSandro2000) [inline pkgs/tools/audio/kaldi/default.nix:71]:
```suggestion
    pkg-config
```

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 108027` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>kaldi</li>
  </ul>
</details>

```
CMake Error at /nix/store/fhp2mjmbhkjsn3czhd0y8fphs83xxnq1-cmake-3.19.1/share/cmake-3.19/Modules/FindPackageHandleStandardArgs.cmake:218 (message):
  Could NOT find BLAS (missing: BLAS_LIBRARIES)
Call Stack (most recent call first):
  /nix/store/fhp2mjmbhkjsn3czhd0y8fphs83xxnq1-cmake-3.19.1/share/cmake-3.19/Modules/FindPackageHandleStandardArgs.cmake:577 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindBLAS.cmake:892 (find_package_handle_standard_args)
  CMakeLists.txt:89 (find_package)


-- Configuring incomplete, errors occurred!
See also "/tmp/nix-build-kaldi-2020-12-26.drv-0/source/build/CMakeFiles/CMakeOutput.log".
See also "/tmp/nix-build-kaldi-2020-12-26.drv-0/source/build/CMakeFiles/CMakeError.log".
```

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review. If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 108027` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>kaldi</li>
</ul>
</details>

**Comment by Mic92** [comment]
Now it only builds for Linux. It probably would build for macOS too, but it requires some blas fixes.

**Review by mweinelt — APPROVED** [review]

**Pull request merged**